src/exercises/7-Naive_Modal/Propmpt.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Input from '../../components/primitives/Input';
+import Button from '../../components/primitives/Button';
+import Heading from '../../components/primitives/Heading';
+import Stack from '../../components/layout/Stack';
```

Comment:
Lets not use any of those and just HTML since that's what we expect from participants.

Reply:
Didn't we agreed on introducing primitives with a short exercise _before_ this one?